variables.tf

```diff
@@ -71,6 +71,12 @@ variable "environment" {
   type = string
 }
 
+variable "fedora_ssl" {
```

Comment:
I think this variable name could be mistaken for "enable ssl for avalon->fedora connection". May I suggest `fcrepo_db_ssl`, it's less ambiguous and more inline with other variables?

Reply:
Sounds good. I'll make the change